src/HttpRequest.cc

```diff
@@ -738,3 +738,62 @@ HttpRequest::manager(const CbcPointer<ConnStateData> &aMgr, const AccessLogEntry
     }
 }
 
+/// a helper for validating FindListeningPortAddress()-found address candidates
+static const Ip::Address *
+FindListeningPortAddressInAddress(const Ip::Address *ip)
+{
+    // FindListeningPortAddress() callers do not want "any" addresses
```

Comment:
s/"any"/INADDR_ANY/

Reply:
@amishxda, please change. I want to minimizing pushing into your branch...

Reply:
Done.